Sources/SwiftDocC/Infrastructure/DocumentationConverter.swift

```diff
@@ -377,6 +377,40 @@ public struct DocumentationConverter: DocumentationConverterProtocol {
         // Log the peak memory.
         benchmark(add: Benchmark.PeakMemory())
 
+        if LinkResolutionMigrationConfiguration.shouldReportLinkResolutionPathMismatches {
+            if context.linkResolutionMismatches.pathsWithMismatchedDisambiguation.isEmpty {
+                print("All symbol paths have the same disambiguation information in both link resolver implementations.")
```

Comment:
Might be good to prefix these with some marker so that we can easily spot them manually and in automation, like `[HierarchyBasedLinkResolutionDiff]`

Reply:
I added a prefix for each type of reported mismatch but not for each individual line.